src/webots/scene_tree/WbSceneTree.cpp

```diff
@@ -728,10 +728,12 @@ void WbSceneTree::convertProtoToBaseNode(bool rootOnly) {
     while (it.hasNext()) {
       it.next();
       const QString destination(WbProject::current()->worldsPath() + it.key());
-      const QFileInfo fileInfo(destination);
-      if (!QDir(fileInfo.absolutePath()).exists())
-        QDir().mkpath(fileInfo.absolutePath());
-      QFile::copy(it.value(), destination);
+      if (!(destination.contains("webots://") || destination.contains("http://") || destination.contains("https://"))) {
```

Comment:
I think it is better to use the functions in `WbUrl` class rather than hard coding again all the web prefixes.
This will make the code easier to maintain.

Moreover, it seems enough to check the `it.key()` prefix instead of the whole `destination` value.
So at least the check for HTTP and HTTPS could be replaced by the `WbUrl::isWeb(it.key())` function.

Then, it would also be better to also have a single function in `WbUrl` class that checks for URLs  starting with `webots://` (currently this prefix is hard coded and checked in 4 different classes).